types/react-native-communications/index.d.ts

```diff
@@ -5,7 +5,7 @@
 // TypeScript Version: 2.3
 
 export function phonecall(phoneNumber: string, isPrompt: boolean): void;
-export function email(to: string, cc: string, bcc: string, subject: string, body: string): void;
+export function email(to?: string[], cc?: string[], bcc?: string[], subject?: string, body?: string): void;
```

Comment:
this doesn't match the docs:

>You must supply either 0 arguments or the full set (5).


I think you should just add an overload with no-arguments, instead of making them optional.

Reply:
Oh right. I'm new to ts and didn't realize it had overload which is clearly a far better option. Thanks!